### ISSUE TITLE
add note on why use of unsafePerformEffect is safe

### ISSUE THREAD

**Issue by safareli** [paf31/purescript-event#3]
When you first look at definition of fix it's not clear why use of unsafePerformEffect is safe, would be great if an explanation is added

**Comment by paf31** [comment]
Agreed, I'll be happy to merge a PR which adds a note along these lines.

**Comment by safareli** [comment]
Use of unsafePerformEffect is safe, as it's effects are not observable (i tried but i couldn't), but I can't really prove or explain well why it's like that.
